tests/workloads/aiconfig/test_aiconfigurator_sdk_contract.py

```
@@ -0,0 +1,217 @@
+def _fetch_pinned_sdk_src() -> Path:
+    """``uv pip install --no-deps`` the pinned aiconfigurator into a temp dir; return its ``sdk`` path.
+
+    ``--no-deps`` skips the entire torch/cuda tree, so this is seconds and tiny. We
+    never import the package (its compute modules need torch); callers AST-parse it.
+    """
+    uv = shutil.which("uv")
+    if uv is None:
+        raise RuntimeError("uv is not available to fetch the pinned aiconfigurator source")
+
+    target = Path(tempfile.mkdtemp(prefix="aiconfigurator_sdk_"))
+    proc = subprocess.run(
```

Ruff (0.15.18)

[error] `subprocess` call: check for execution of untrusted input

(S603)